packages/app/src/app/overmind/namespaces/live/liveMessageOperators.ts

```diff
@@ -10,21 +9,23 @@ import { Operator } from 'app/overmind';
 import { camelizeKeys } from 'humps';
 import { NotificationStatus } from '@codesandbox/notifications/lib/state';
 
-export const onJoin: Operator<LiveMessage<LiveMessageEvent.JOIN>> = mutate(
-  ({ effects, state }) => {
-    effects.notificationToast.success(
-      state.live.isTeam ? 'Connected to Live Team!' : 'Connected to Live!'
-    );
+export const onJoin: Operator<
+  LiveMessage<{ status: 'connected'; live_user_id: string }>
```

Comment:
Why remove the Event type? :)

Reply:
I wanted to add payload to the message (`status` and `live_user_id`), is there another way to do this?

Reply:
Ah, `LiveMessageEvent.JOIN` was wrong here... sorry!